src/AddTodoForm.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import InputWithLabel from './InputWithLabel'
+
+function AddTodoForm(props) {
+	const { onAddTodo } = props
+	const [todoTitle, setTodoTitle] = React.useState('')
+
+	function handleTitleChange(event) {
+		const newTodoTitle = event.target.value
+		setTodoTitle(newTodoTitle)
+	}
+
+	function handleAddTodo(event) {
+		event.preventDefault()
+
+		onAddTodo({ title: todoTitle, id: Date.now() })
+		setTodoTitle('')
+	}
+
+	return (
+		<>
```

Comment:
You have an interesting bug in your user interface. This is something that affects the user experience. Play around with your app and see if you notice anything that's not working as expected. Don't spend too much time trying to find the bug so lmk if you spend more than 5 minutes on it. I'd rather you spend time fixing it. Good luck!